en/sql/tuning.rst

```diff
@@ -2337,3 +2340,80 @@ The above **SELECT** query's plan is printed out as below; we can see "(sort lim
                    cost:  6 card 1000
         sort:  2 asc
         cost:  7 card 0
+
+.. _query-cache:
+
+QUERY CACHE
+===========
+
+The **QUERY_CACHE** hint can be used to enhance the performance for the query which is executed repeatedly. The query is cached in dedicated memory area and its results are also cached at the separated disk space. The hint is applied to SELECT query only; however for the following cases, the hint is not applied to the query:
+
+*   a system time or date related attribute in the query as below
+    ex) SELECT SYSDATE, ADDDATE(SYSDATE,INTERVAL -24 HOUR), ADDDATE(SYSDATE, -1);
+*   a serial related attribute is in the query
+*   a column-path related attribute is in the query
+*   a method is in the query
+*   a stored procedure or a stored function is in the query
+*   a system tables like dual, _db_attribute, and so on, is in the query
+*   a system function like sys_guid() is in the query
+
+When the hint is set and a new SELECT query is processed, the query cache is looked up if the query appears in the query cache. The queries are considered identical in case that they use the same query text and the same bind values under the same database. If the cached query is not found, the query will be processed and then cached newly with its result. If the query is found from the cache, the results will be fetched from the cached area. AT the CSQL, we can measure the enhancement easily to execute repeatedly the query using COUNT function as below example. The query and its results will be cached at the first appear, so the response time is slower than the next same query. The second query's result is fetched from cached area, so the response time is very faster than the prior same query's one. ::
```

Comment:
1. The queries are considered identical in case that they..... -->The queries are considered identical in case they....
(delete 'the')

2. we can measure the enhancement easily to execute repeatedly the query using COUNT function as below example.
-->we can measure the enhancement easily to execute the query repeatedly
--> the COUNT function (adding 'the')

3. The query and its results will be cached at the first appear -->
suggestion 1: at the first appearance (verb change to noun)
suggestion 2: at first (at the first appear --> at first)

4. The second query's result is fetched from cached area, so the response time is very faster than the prior same query's one. ::
--> The second query's result is fetched from the cached area (adding 'the')
--> the response time is very faster--> the response time is way faster/ much faster... (if you want to emphasize how fast it is, very should change to 'much' or 'way')

Reply:
Ok, thanks